openpype/hosts/max/api/lib.py

```diff
@@ -128,7 +128,14 @@ def list_children(node):
 
 
 def get_current_renderer():
-    """get current renderer"""
+    """
+    Notes:
+        Get current renderer for Max
+
+    Returns:
+        "{Current Renderer}:{Current Renderer}"
+        e.g. "Redshift_Renderer:Redshift_Renderer"
```

Comment:
> If this always returns the value like that, why wouldn't we split it directly as part of this function? Or do the values ever differ?

nothing different, but it should be the separate PR as it will affect the render instance.